tests/ui/pages/desktop/regions/sorter.py

```diff
@@ -0,0 +1,15 @@
+from selenium.webdriver.common.by import By
+
+from pypom import Region
+
+
+class Sorter(Region):
+    """Helper class for sorting."""
+    _root_locator = (By.ID, 'sorter')
+    _sort_by_type_locator = (By.CSS_SELECTOR, 'ul > li')
+    _updating_locator = (By.CSS_SELECTOR, '.updating')
+
+    def sort_by(self, sort):
+        """Clicks the sort by type requested."""
```

Comment:
is there a word missing? "Clicks the sort button for the requested sort-order" sounds more reasonable, but I don't know if that's what you want to say so feel free to rewrite this.